src/Integration/Connection/UI/OrganizationSelectionWindow.xaml

```diff
@@ -41,46 +42,98 @@
 
     <!--  Design  -->
-    <StackPanel Style="{StaticResource DialogContent}">
-        <TextBlock Margin="0,0,0,8"
-                   FontStyle="Italic" Text="Please select the organization you want to bind to." TextWrapping="Wrap" />
-
-        <StackPanel Orientation="Horizontal">
-            <TextBlock VerticalAlignment="Center" Text="Organizations" />
-            <ComboBox x:Name="OrganizationComboBox"
-                      Width="250" Margin="12,0,0,0"
-                      HorizontalContentAlignment="Stretch"
-                      IsEditable="True" IsTextSearchCaseSensitive="False" IsTextSearchEnabled="True"
-                      TextSearch.TextPath="Name">
-                <ComboBox.ItemsPanel>
-                    <ItemsPanelTemplate>
-                        <VirtualizingStackPanel />
-                    </ItemsPanelTemplate>
-                </ComboBox.ItemsPanel>
-                <ComboBox.ItemContainerStyle>
-                    <Style TargetType="{x:Type ComboBoxItem}">
-                        <Setter Property="Width" Value="230" />
-                    </Style>
-                </ComboBox.ItemContainerStyle>
-                <ComboBox.ItemTemplate>
-                    <DataTemplate DataType="{x:Type models:SonarQubeOrganization}">
-                        <TextBlock TextTrimming="CharacterEllipsis">
-                            <Run Text="{Binding Name, Mode=OneWay}" />
-                            <Run Text="(" /><Run Text="{Binding Key, Mode=OneWay}" /><Run Text=")" />
-                        </TextBlock>
-                    </DataTemplate>
-                </ComboBox.ItemTemplate>
-            </ComboBox>
-        </StackPanel>
-
-        <Separator Style="{StaticResource DialogSeparator}" />
-
-        <StackPanel VerticalAlignment="Center" Style="{StaticResource DialogButtonsPanel}">
-            <vsui:DialogButton Click="OnOkButtonClick" Content="OK" IsDefault="True"
-                               IsEnabled="{Binding ElementName=OrganizationComboBox,
-                                                   Path=SelectedItem,
-                                                   Converter={StaticResource NotNullConverter}}" />
-            <vsui:DialogButton Content="Cancel" IsCancel="True" />
-        </StackPanel>
-    </StackPanel>
+    <!-- Note: we're using grids inside the TabItem with IsSharedSizeScope to make sure the tabs are the same size
+         (otherwise they will be different and the dialogue box will resize when the user switches tabs) -->
+    <TabControl Grid.IsSharedSizeScope="True" >
+        <TabItem Header="Your organizations">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition SharedSizeGroup="tabItemSize" />
+                </Grid.RowDefinitions>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition SharedSizeGroup="tabItemHeight" />
+                </Grid.ColumnDefinitions>
+
+                <StackPanel 
```

Comment:
This contents of this `StackPanel` are essentially the old UI with the combobox. I haven't changed it at all; all I've done is move it inside a tab labelled `Your organizations`.